styles/index.css

```diff
@@ -0,0 +1,116 @@
+:root {
+    --main-bg-color: rgb(16, 3, 46);
+    --starwars-yellow: rgb(88%, 69%, 0%);
```

Comment:
I am aware of the naming convention, (to avoid using adjectives in names), but in this context yellow is  just constant, fetched from SW logo. However its not a big deal to change this if its needed.

Reply:
I see. It can stay like this.